### PR TITLE
WIP: Feat   mihaelachetran]  pug scss] large image with tall and normal article block #19

### PR DESCRIPTION
**Pull request by mihaelachetran**

### CODE REVIEW

File: pug-sass/src/mihaelachetran/index.pug [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```
Comment (marianzburlea) [inline pug-sass/src/mihaelachetran/index.pug:1]: update doctype with all lowercase

File: pug-sass/src/mihaelachetran/index.pug [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+html(lang="en")
+  head
+    meta(charset="UTF-8")
+    meta(name="viewport", content="width=device-width, initial-scale=1.0")
+    title  Mihaela's home solutions
```
Comment (marianzburlea) [inline pug-sass/src/mihaelachetran/index.pug:6]: two spaces

File: pug-sass/src/mihaelachetran/index.pug (hunk 1 above)
Comment (mihaelachetran) [reply pug-sass/src/mihaelachetran/index.pug:1]: Done!
doctype html

File: pug-sass/src/mihaelachetran/index.pug (hunk 2 above)
Comment (mihaelachetran) [reply pug-sass/src/mihaelachetran/index.pug:6]: Done!
Thank you!

File: pug-sass/src/mihaelachetran/blog-blocks/index.pug [hunk 3]
```diff
@@ -1,16 +1,24 @@
...
-  body
-    .main-box
-      .images-container
-        h1.images-title Effective Forms Advertising Internet Web Site
-        p.images-subtitle.images-subtitle Make Money Online Through Advertising
-        .images-avatar
-        .images-name Lora Steven
-        .images-date Aug 15
-        .images-icon
+head
```
Comment (marianzburlea) [inline pug-sass/src/mihaelachetran/blog-blocks/index.pug:3]: head and body have to be children of the **html** tag

File: pug-sass/src/mihaelachetran/blog-blocks/index.scss [hunk 4]
```diff
@@ -20,37 +21,82 @@ body {
...
 
 .images-avatar {
+  background-image: url(/asset/image/mihaelachetran/avatar.png);
+}
+
+.images-avatar2 {
+  background-image: url(/asset/image/mihaelachetran/avatar2.png);
+}
+
+.images-avatar, .images-avatar2 {
```
Comment (marianzburlea) [inline pug-sass/src/mihaelachetran/blog-blocks/index.scss:85]: separate selectors on new line

File: pug-sass/src/mihaelachetran/blog-blocks/index.pug (hunk 3 above)
Comment (mihaelachetran) [reply pug-sass/src/mihaelachetran/blog-blocks/index.pug:3]: Thank you! Done!

File: pug-sass/src/mihaelachetran/blog-blocks/index.scss (hunk 4 above)
Comment (mihaelachetran) [reply pug-sass/src/mihaelachetran/blog-blocks/index.scss:85]: Done it! Thank you!

**Pull request merged**